packages/orbit-components/transforms/tokens-v16.mjs

```diff
@@ -11,11 +11,41 @@ const transformations = [
   { from: "borderRadiusNormal", to: "borderRadius100" },
   { from: "borderRadiusLarge", to: "borderRadius150" },
   { from: "borderRadiusCircle", to: "borderRadiusFull" },
-  // Add more transformations as needed
 ];
 
+const headingTransformations = [
+  { from: "fontSizeHeadingDisplay", to: "headingDisplayFontSize" },
+  { from: "fontSizeHeadingDisplaySubtitle", to: "headingDisplaySubtitleFontSize" },
+  { from: "fontSizeHeadingTitle1", to: "headingTitle1FontSize" },
```

Comment:
I wonder if this should use Title0 as the replacement already? 🤔

Reply:
We should. Thank you